.github/workflows/main.yml

```diff
@@ -91,23 +91,26 @@ jobs:
 
       - name: Integration tests with pytest
         env:
-          FHIR_API_URL: ${{ secrets.FHIR_API_URL }}
           FHIR_API_TOKEN: ${{ secrets.FHIR_API_TOKEN }}
-          PYROG_API_URL: ${{ secrets.PYROG_API_URL }}
-          PYROG_LOGIN: ${{ secrets.PYROG_LOGIN }}
-          PYROG_PASSWORD: ${{ secrets.PYROG_PASSWORD }}
-          MIMIC_PRACTITIONER_RESOURCE_ID: ${{ secrets.MIMIC_PRACTITIONER_RESOURCE_ID }}
-          MIMIC_PATIENT_RESOURCE_ID: ${{ secrets.MIMIC_PATIENT_RESOURCE_ID }}
-          KAFKA_BOOTSTRAP_SERVERS: ${{ secrets.KAFKA_BOOTSTRAP_SERVERS }}
-          MIMIC_DB_PASSWORD: ${{ secrets.MIMIC_DB_PASSWORD }}
+          AUTH_SIGNING_PUBLIC_KEY: ${{ secrets.AUTH_SIGNING_PUBLIC_KEY }}
+          MIMIC_PRACTITIONER_RESOURCE_ID: ckbjjts9k008758pwuo4uupux
```

Comment:
I would query pyrog to get all the resource of all the source directly in the integration test in order to avoid defining these env variables